lib/widgets/payment_details_dialog.dart

```diff
@@ -78,23 +78,32 @@ Future<Null> showPaymentDetailsDialog(
                           ? 16
                           : 8),
                   child: AutoSizeText(
-                    paymentInfo.dialogTitle,
+                    paymentInfo.dialogTitle.replaceAll("\n", " "),
                     style: Theme.of(context).primaryTextTheme.headline5,
                     textAlign: TextAlign.center,
+                    overflow: TextOverflow.ellipsis,
                     maxLines: 1,
                   ),
                 ),
           paymentInfo.description == null || paymentInfo.description.isEmpty
               ? Container()
               : Padding(
                   padding: EdgeInsets.only(left: 16.0, right: 16.0),
-                  child: AutoSizeText(
-                    paymentInfo.description,
-                    style: Theme.of(context).primaryTextTheme.headline4,
-                    textAlign: paymentInfo.description.length > 40
-                        ? TextAlign.justify
-                        : TextAlign.center,
-                    maxLines: 3,
+                  child: Container(
+                    constraints: BoxConstraints(
+                        maxHeight: 54, minWidth: double.infinity),
+                    child: Scrollbar(
+                      child: SingleChildScrollView(
+                        child: AutoSizeText(
+                          paymentInfo.description,
+                          style: Theme.of(context).primaryTextTheme.headline4,
+                          textAlign: paymentInfo.description.length > 40 &&
```

Comment:
Why is the text alignment related to the text length?

Reply:
It's for long texts that overflow to the next line.
[abc...........uvw]
[..........xyz..........]
->
[abc...........uvw]
[xyz....................]